chat-plugins/calculator.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+function isNumeric(str) {
+	return !isNaN(parseFloat(str));
```

Comment:
Considering this is short and only used once, I'd probably recommend inlining this function.

Reply:
done